src/main/resources/application.yml

```diff
@@ -3,9 +3,61 @@ eureka:
     serviceUrl:
       defaultZone: http://localhost:8761/eureka/
   instance:
-    instanceId: ${spring.application.name}:${random.value}
+    instanceId: ${spring.application.name}
 server:
   port: 8095
 spring:
+  data:
+    authService: auth-service
+  url:
+    authServiceUserValidationPath: /v1/validating-token
   application:
     name: api-gateway
+
+#Routing with zuul
+zuul:
+  routes:
+    auth-token-generation:
+      path: /v1/authenticate
+      serviceId: auth-service
+      stripPrefix: false
+    auth-token-validation:
+      path: /v1/validating-token
+      sensitiveHeaders: Cookie,Set-Cookie
```

Comment:
curious, as to why is this header used for ?

and why we dont have that in the url above

Reply:
Removed,  By default, headers Cookie, Set-Cookie, Authorization are blocked by the sensitiveHeaders: Cookie,Set-Cookie,Authorization configuration.

Reply:
We can share headers between services in the same system, but we probably do not want sensitive headers leaking downstream into external servers